crt_portal/cts_forms/migrations/0087_expand_form_subject_length.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.13 on 2020-08-18 15:53
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cts_forms', '0086_materialize_trends_view'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='report',
+            name='assigned_section',
+            field=models.TextField(choices=[('ADM', 'Administrative'), ('APP', 'Appellate'), ('CRM', 'Criminal'), ('DRS', 'Disability Rights'), ('ELS', 'Employment Litigation'), ('EOS', 'Educational Opportunities'), ('FCS', 'Federal Coordination and Compliance'), ('HCE', 'Housing and Civil Enforcement'), ('IER', 'Immigrant and Employee Rights'), ('SPL', 'Special Litigation'), ('VOT', 'Voting')], default='ADM'),
```

Comment:
This alter field change isn't necessary and doesn't really do anything to the database (we don't change the section choice keys), but the auto-migration picked it up.

Reply:
Good catch!

We should consider adding this to our CI pipeline, https://docs.djangoproject.com/en/2.2/ref/django-admin/#cmdoption-makemigrations-check